### DISCUSSION TITLE
Issues with specific SATA drive

### DISCUSSION THREAD

**Discussion by derritter88** [openzfsonwindows/openzfs#343] [General]
I have tried to use ZFS on a Seagate IronWolf 8 TB NAS drive - according to infos found on the internet it's a CMR drive.
When doing any backup jobs via FreeFileSync I get a lot of I/O errors via `zpool status` and the device get's suspended.

Earlier I was using this drive with VeraCrypt + NTFS without any issues.

On my other backup drives (SATA & SAS) I do not experience any of those issues.

IMHO the drive itself is not damaged as I am currently using it again with VeraCrypt and can copy stuff over it without any issues.

Anyone got an idea what might be wrong?

**Comment by EchterAgo** [comment]
You might want to check windows error logs (Win+X V, `Windows Logs | System`) and if any SMART attributes increase.

FWIW, I'm using Windows ZFS with the 4TB IronWolf NAS models without any issues.

**Reply by derritter88** [reply]
I check and SMART errors via hwinfo and according to them the drive is fine

**Reply by EchterAgo** [reply]
I don't mean the drives health self assessment but actually monitoring the attributes. For example UDMA CRC error count can increase without failing health and it usually indicates cabling.

**Reply by derritter88** [reply]
It does not show any issues. I just checked the Event Viewer if there are any issues but there are nothing.

**Comment by guenther-alka** [comment]
Ntfs does not have checksums so it cannot detect all errors like ZFS can do.
If ZFS reports problems there are problems.

If the same disk previously worked in another setup, I would first replace Sata cables as this could be the reason. 
Next I would do an intensive disk test ex via WD datalifguard  (ex from a Hirens USB bootstick with Windows 10PE) .

https://www.hirensbootcd.org/

**Comment by derritter88** [comment]
SATA cable should not be a problem as all other drives works well.
I downloaded a Seagate software for testing the drive as it is an IronWolf.

**Comment by derritter88** [comment]
I ran a "long generic test" which took 0.5 day to complete - no errors were detected.

**Comment by derritter88** [comment]
Interessting as after removing the previous Veracrypt encrypted NTFS filesystem and reformatting it with ZFS + native ZFS encryption currently it works without any I/O errors.
Will keep watching it.
